keras_cv/losses/ciou_loss_test.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.losses.ciou_loss import CIoULoss
+
+
+class CIoUTest(tf.test.TestCase):
+    def test_output_shape(self):
+        y_true = tf.random.uniform(
+            shape=(2, 2, 4), minval=0, maxval=10, dtype=tf.int32
+        )
+        y_pred = tf.random.uniform(
+            shape=(2, 2, 4), minval=0, maxval=20, dtype=tf.int32
+        )
+
+        ciou_loss = CIoULoss(bounding_box_format="xywh")
+
+        self.assertAllEqual(ciou_loss(y_true, y_pred).shape, ())
+
+    def test_output_shape_reduction_none(self):
+        y_true = tf.random.uniform(
+            shape=(2, 2, 4), minval=0, maxval=10, dtype=tf.int32
+        )
+        y_pred = tf.random.uniform(
+            shape=(2, 2, 4), minval=0, maxval=20, dtype=tf.int32
+        )
+
+        ciou_loss = CIoULoss(bounding_box_format="xyxy", reduction="none")
+
+        self.assertAllEqual(
+            [2, 2],
+            ciou_loss(y_true, y_pred).shape,
+        )
+
+    def test_output_shape_relative_formats(self):
```

Comment:
I'm not sure the output shape assertion is particularly meaningful here, just testing a relative format, right?

So maybe we should parameterize the `output_value` test to run as both absolute and relative format, then we can assert that the losses match

(e.g. scale the y_true and y_pred by 1/640 or similar, then say format = "rel_xyxy" and assert the value is the same)